gradio/events.py

```diff
@@ -90,7 +95,7 @@ def __init__(
 
     def __call__(
         self,
-        fn: Callable | None,
+        fn: Callable | None | Literal["wrapper"] = "wrapper",
```

Comment:
Just wondering, where is this "wrapper" coming from? I don't see it anywhere else in the code.

Reply:
it's just the default value - if you don't set it, it acts like a decorator. There's no way for a function to know if it's being used as a decorator, so if you don't set fn, we assume it's a decorator. Previously fn had no default value so this is not a breaking change.

Reply:
also renamed to "decorator" to be clearer